doc/import_export.rst

```diff
@@ -17,78 +14,46 @@ Create a folder with CSVs for each component type
 time-dependent variable (e.g. ``generators-p_max_pu.csv,
 loads-p_set.csv``).
 
-Then run
-
-``network.import_from_csv_folder(csv_folder_name)``
+Then run ``network.import_from_csv_folder(csv_folder_name)``.
 
-See the :doc:`examples` in pypsa/examples/.
+.. autofunction:: pypsa.io.import_from_csv_folder
```

Comment:
This is rendered as `pypsa.io.import_from_csv_folder(network, csv_folder_name, encoding=None, skip_time=False)`, since we hook it into the `Network` object, it might be better to use `automethod`, which would render as
```
Network.import_from_csv_folder(network, csv_folder_name, encoding=None, skip_time=False)
```
wouldn't it?